doc/wg/README.md

```diff
@@ -1,9 +1,109 @@
 Tock Working Groups
 ===================
 
-Working groups (wg) are focused groups to organize development around a
+Working groups are focused groups to organize development around a
 particular aspect of Tock.
 
+## Motivation
+
+Tock relies on contributions from developers with across a variety of
+organizations, physical distance, timezones, and engagement. It has
+also grown large in scope. This has resulted in strain on maintainers
+to regularly keep up with contributions while ensuring quality and
+important global policies like safety and security. It has also
+resulted in contributors' work going unaddressed for long periods of
+time.
```

Comment:
I think this text is helpful in this PR description, but the goal is for this to become incorrect, which makes it awkward to add to the README. I think either we don't need it or it could be phrased as what we want to avoid, rather than what is currently happening.